/**:
  ros__parameters:
    device: "cuda"
    object_frame_id: "pylone_link"
    world_frame_id: "fer_link0"
    detector_model_id: "openmmlab-community/mm_grounding_dino_base_all"
    megapose: 
      megapose_model_config: "megapose-1.0-RGB-multi-hypothesis-icp"
      mesh_megapose_relative_path: "assets/meshes/pylone_but_better.obj"
      SO3_grid_size_scale_down: 2
    s2m2: 
      model_type: "S"
      allow_negative: false
      num_refine: 3
      torch_compile: false
    icp:
      use_different_mesh_for_icp: true
      mesh_icp_relative_path: "assets/meshes/pylone_icp_metal_2parts.obj"
      margin_sphere_crop: 1.1
      voxel_size: 0.005
      dist_thresh_factor: 3.0
      icp_method: "point2plane"
      reject:
        translation_th: 0.05
        rotation_th: 10